src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderPersonRestricts.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType.LAZY
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.JoinColumnOrFormula
+import org.hibernate.annotations.JoinColumnsOrFormulas
+import org.hibernate.annotations.JoinFormula
+import java.time.LocalDate
+
+@Entity
+@Table(name = "OFFENDER_PERSON_RESTRICTS")
+data class OffenderPersonRestricts(
+  @Id
+  @SequenceGenerator(
+    name = "OFFENDER_PERSON_RESTRICT_ID",
+    sequenceName = "OFFENDER_PERSON_RESTRICT_ID",
+    allocationSize = 1,
+  )
+  @GeneratedValue(generator = "OFFENDER_PERSON_RESTRICT_ID")
+  @Column(name = "OFFENDER_PERSON_RESTRICT_ID", nullable = false)
+  val id: Long = 0,
+
+  @ManyToOne(optional = false, fetch = LAZY)
+  @JoinColumn(name = "OFFENDER_CONTACT_PERSON_ID", nullable = false)
+  val contactPerson: OffenderContactPerson,
+
+  @ManyToOne(optional = false, fetch = LAZY)
+  @JoinColumn(name = "ENTERED_STAFF_ID", nullable = true)
+  val enteredStaff: Staff,
```

Comment:
Is nullable correct?

Reply:
yes, the column is nullable but none for the data is null - so I am using this mixture